### PR TITLE
Add `Authorization Code Flow` to example app

### PR DESCRIPTION
**Pull request by micseltz-cisco**
Added the `Authorization Code Flow` from https://developer.webex.com/docs/login-with-webex

Improved readme and added variables to be configured as environment vars

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,18 @@ https://developer.webex.com/docs/login-with-webex
 
 Try it out by chosing one of the available flows: https://webexsamples.github.io/login-with-webex
 
+To start the server run:
+
+`npm start`
+
+Optional parameters are:
+WEBEX_BASE=https://integration.webexapis.com
```
Comment (johnwfrancis) [inline README.md]: I'd format these guys as an unordered list and enclose the param names in backticks.

**Review by johnwfrancis — COMMENTED** [review]
Optional changes.

File: README.md (hunk 1 above)
Comment (micseltz-cisco) [reply README.md]: thanks for the review - updated!

Comment (Joezanini) [inline README.md]: Please change https://integration.webexapis.com to https://webexapis.com according to the [docs](https://developer.webex.com/docs/api/v1/authorizations) for non-cisco users.

Comment (Joezanini) [reply README.md]: I also noticed that the authorization code flow ends by displaying the authorization code. Is this the intended end of the flow? should we add functionality that shows user info?
Screenshot 2024-10-22 at 2 07 52 PM

Comment (micseltz-cisco) [reply README.md]: Sure can change the example. For my use case it was helpful to have the code unformatted, feel free to change this how you see fit if you think there's something more generically useful

**Comment by micseltz-cisco** [comment]
Closing for now after discussion with @Joezanini, pointed me towards https://github.com/WebexSamples/webex-oauth-integration insteadm if there are pieces of this that are helpful around environmental variables and such feel free to pick anything

**Pull request closed without merging**